openverse_catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -0,0 +1,174 @@
+"""
```

Comment:
Note for reviewers: This file just pulls out all the old data_refresh steps into a new subfactory.